src/modopt/signal/validation.py

```diff
@@ -36,6 +37,8 @@ def transpose_test(
         Shape of transpose operator input data (default is ``None``)
     y_args : tuple, optional
         Arguments to be passed to transpose operator (default is ``None``)
+    rng: np.random.Generator or int or None (default is ``None``)
```

Comment:
I guess this will need to be `numpy.random.Generator` for the intersphinx mapping to work.